### ISSUE TITLE
"Loading" image missing alt attribute

### ISSUE THREAD

**Issue by brockfanning** [open-sdg/open-sdg#10]
This error is appearing in htmlproofer: `image /sdg-indicators/assets/img/loading.gif does not have an alt attribute`

**Comment by brockfanning** [comment]
False alarm. This was being caused by a failing connection with the translation repository.
